radis/lbl/factory.py

```diff
@@ -1063,8 +1058,17 @@ def eq_spectrum_gpu(
         Other Parameters
         ----------------
         backend: str
-            if ``'cuda-gpu'``, set CUDA as backend to run code on Nvidia GPU.
-            if ``'cuda-cpu'``, execute the GPU code on the CPU (useful for development)
+            currently only ``'vulkan'`` backend is supported.
+            ``'gpu-cuda'`` and``'cpu-cuda'`` used to be available to switch to a CUDA backend,
+            but have been deprecated in favor of the Vulkan backend.
+        device_id: int, str
+            Select the Vulkan device. If ``int``, specifies the device index, which is printed when Vulkan is loaded.
```

Comment:
```suggestion
            Select the GPU device. If ``int``, specifies the device index, which is printed when Vulkan (the GPU backen d) is loaded.
```

added cleared message because users are not necessarily aware of what Vulkan is